Patch-Source: https://github.com/gnuradio/volk/pull/869

From 5620097efb4a70620259000d27918dee1d03ee1e Mon Sep 17 00:00:00 2001
From: Johannes Sterz Demel <jdemel@gnuradio.org>
Date: Sun, 28 Jun 2026 15:25:48 +0200
Subject: [PATCH] fmt: Switch include from core to format

Fixes a breaking change introduced in fmtlib 12.2.0.

https://github.com/fmtlib/fmt/releases/tag/12.2.0

> Made the <fmt/core.h> header equivalent to <fmt/base.h> by default. Code that relied on <fmt/core.h> pulling in <fmt/format.h> must now either include <fmt/format.h> directly or define FMT_DEPRECATED_HEAVY_CORE to opt back in.

Signed-off-by: Johannes Sterz Demel <jdemel@gnuradio.org>
---
 lib/qa_utils.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/qa_utils.cc b/lib/qa_utils.cc
index b336c1d22..c635f3e7f 100644
--- a/lib/qa_utils.cc
+++ b/lib/qa_utils.cc
@@ -28,7 +28,7 @@
 #include <random>
 #include <vector> // for vector, _Bit_refe...
 
-#include <fmt/core.h>
+#include <fmt/format.h>
 
 // Warmup time for CPU frequency scaling (ms)
 static double g_warmup_ms = 2000.0;
